src/components/header/header.js

```diff
@@ -16,9 +16,11 @@ export default class Header extends HTMLElement {
       this.attachShadow({ mode: "open" });
       this.shadowRoot.innerHTML = `
         <header>
+        <a href="/" title="Greenwood Logo">
```

Comment:
A couple thoughts here:
1. For the title of this link, I think we might probably want it to say _Greenwood home page_ instead
1. I _think_ we might want to wrap the `<a>` tag _inside_ the div, e.g.
    ```html
    <div class="logo-bar">
      <a href="/" title="Greenwood home page">
        ${greenwoodLogo}
      </a>
    </div>
    ```

Reply:
@thescientist13  sometimes, we often wrap clickable elements like cards, buttons with <a> tags just like below screenshot:

![image](https://github.com/ProjectEvergreen/www.greenwoodjs.dev/assets/135650558/e32bb8e3-2598-4dff-a9b8-d08bd4408a7a)

Reference: https://developer.mozilla.org/en-US/curriculum/. Let me now if that's Okay. 
I will proceed with other changes. Thanks!

Reply:
Yeah more so just confirming  [from this article](https://www.bennadel.com/blog/3693-anchor-tags-can-contain-block-level-elements-as-of-html5.htm) that `<a>` tags can contain block level elements within them, so seems like what you have here should be fine.  👍